proto/README.md

```diff
@@ -56,10 +63,11 @@ yaw | radians | float | Bounding box rotation angle along the Z axis.
 
 Field | Unit | Type | Description
 --- | --- | --- | ---
-positions | meters | Vector3 (list) | List of object’s XYZ position.
+positions | meters | Vector3 (list) | List of object’s tracked XYZ position.
 
 ### Prediction
 
 Field | Unit | Type | Description
 --- | --- | --- | ---
-positions | meters | Vector3 (list) | List of object’s XYZ position.
+positions | meters | Vector3 (list) | List of object’s predicted XYZ position.
+reachable_set | meters | [ReachableSet](sensr_proto/type.proto) | List of object's predicted range which includes uncertainty sorted by time-lapse.
```

Comment:
What does "sorted by time-lapse mean"? Is this supposed to be something like "sorter by time elapsed"?

Reply:
@DK-sr Could you check Thor's suggestion?

Reply:
agree with Thor. Maybe "List of object's predicted position including uncertainty sorted by elapsed time." is the best description?

Reply:
Yup, I agree with both Thor and Donny.

Reply:
@DonnyKim-SeoulRobotics @DK-sr  Could you last-check? Is "position" fine in this sentence?

Reply:
I feel like position explains it better. How do you think @thor-sr ?

Reply:
for me looks good. https://youtu.be/U9d9hwU2RIc?t=51

Reply:
yup, looks good to me too